modules/mockserver/mockserver_test.go

```diff
@@ -0,0 +1,26 @@
+package mockserver
+
+import (
+	"context"
+	"testing"
+
+	"github.com/testcontainers/testcontainers-go"
+)
+
+func TestMockServer(t *testing.T) {
```

Comment:
We can remove this test, as it's not doing any assertion.

I have opposed sentiments when writing both testable examples and tests: what is a testable example and what is a test? In general, I like testable examples because they end up into the Go docs. But I do prefer not adding too many testable examples and expand the test suite as regular tests. Wdyt?

Reply:
I see no value in this test specifically because the example already covers this use case. In general, I prefer to avoid using examples as testing because you can end up sacrificing test reliability or example readability. So I prefer to have them separated, but for this specific case where this is so simple and so general, there is no point in adding this test instead of just relying on the example.